### PR TITLE
TINY-9180: Improve discoverability of the question mark read more link

### PR DESCRIPTION
**Pull request by shanmen-tiny**
Related Ticket: TINY-9180

Description of Changes:
* Setting icon alignment to be middle 
* Decrease font size of 'issue' and 'warning/error'
* Remove borders around descriptions
* Set warning and success title colours to meet the contrast ratio
* Setting the background colour of the 'question mark' button

Pre-checks:
* [x] ~~Changelog entry added~~ (Do we need changelog entry for changes made to premium plugin?)
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

Screenshot of UI changes: 
- The `info` state is to demo

**Light Mode:**
SCR-20221013-b5l
SCR-20221013-ayn
SCR-20221013-ayx
SCR-20221013-az3

**Dark Mode:**
SCR-20221013-b5b
SCR-20221013-b3q
SCR-20221013-b3u
SCR-20221013-b41

GitHub issues (if applicable):

### CODE REVIEW

**Comment by lnewson** [comment]
For these sort of changes, it's often a good idea to include a screenshot to show how others it looks in the PR description.

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less [hunk 1]
```diff
@@ -108,21 +108,29 @@
         fill: @accessibility-issue-warn-heading-text-color;
       }
 
-      a .tox-icon {
-        color: @accessibility-issue-warn-heading-text-color;
+      /* stylelint-disable-next-line no-descending-specificity */
```
Comment (ephox-mogran) [inline modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:111]: Should we just nest it inside `a.tox-button--naked` so that we don't have a disable a linting rule. Is this really that much easier to parse/read? Is that what the rule is complaining about?

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less [hunk 2]
```diff
@@ -47,12 +48,11 @@
         align-items: center;
         display: flex;
         margin-bottom: @pad-xs;
-      }
-    }
 
-    > *:last-child:not(:only-child) {
-      border-color: @border-color;
-      border-style: solid;
+        .tox-icon svg {
```
Comment (ephox-mogran) [inline modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:52]: I probably didn't expect this much of a change, but I also haven't looked into it. What's the reason for this change?

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less [hunk 3]
```diff
@@ -83,21 +78,26 @@
         fill: @accessibility-issue-info-heading-text-color;
       }
 
-      a .tox-icon {
-        color: @accessibility-issue-info-heading-text-color;
+      a.tox-button--naked.tox-button--icon {
```
Comment (ephox-mogran) [inline modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:81]: Why does this one not need to disable the linting rule?

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 2 above)
Comment (shanmen-tiny) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:52]: This is to set the icons to be middle aligned. The `svg` defaults to `inline`, it has an extra space which makes the icon a bit off.

Current: 
image

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 1 above)
Comment (TheSpyder) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:111]: I think it doesn't complain about `info` because it's first. If you swapped the order and put something else first then `info` would get the warning too.

Likely the only way to work around the warning is specify the 3 button colours and then repeat the 3 button selectors to set hover/focus/active later in the file. Given we know `error`/`warning`/`info` won't be used at the same time I guess suppressing the warning is fine.

**Review by TheSpyder — APPROVED** [review]

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 2 above)
Comment (ephox-mogran) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:52]: I wonder if we should split that into a separate ticket. This is a different way from how it is addressed in the notification banners, but maybe that doesn't matter. There, they seem to use grid and align-self etc. I guess it all depends on how many other `tox-icon`s we have around with this problem.

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 1 above)
Comment (ephox-mogran) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:111]: Fair enough. Withdrawn.

**Review by ephox-mogran — APPROVED** [review]
